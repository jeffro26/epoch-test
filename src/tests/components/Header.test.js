import React from "react";

import HeaderBar from "../../components/Header";
import renderer from "react-test-renderer";

describe("Header Bar", () => {
  it("Snapshot match", () => {
    const tree1 = renderer.create(<HeaderBar />).toJSON();
    expect(tree1).toMatchSnapshot();
  });
});
