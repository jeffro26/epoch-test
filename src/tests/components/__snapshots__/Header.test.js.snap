// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Bar Snapshot match 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-Bar-2 MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <span
        aria-hidden={true}
        aria-label="menu"
        className="material-icons MuiIcon-root makeStyles-icon-3"
        edge="start"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.24 7.76C15.07 6.59 13.54 6 12 6v6l-4.24 4.24c2.34 2.34 6.14 2.34 8.49 0 2.34-2.34 2.34-6.14-.01-8.48zM12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
          />
        </svg>
      </span>
      <h6
        className="MuiTypography-root makeStyles-title-4 MuiTypography-h6"
      >
        Epoch Site
      </h6>
    </div>
  </header>
</div>
`;
